CCContinuousGlucose/Classes/ContinuousGlucoseAnnunciation.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ContinuousGlucoseAnnunciation.swift
+//  Pods
+//
+//  Created by Kevin Tallevi on 4/19/17.
+//
+//
+
+import Foundation
+import CCToolbox
+
+public class ContinuousGlucoseAnnunciation : NSObject {
+    public var sessionStopped: Bool?
```

Comment:
These variables match the names given in the Bluetooth spec, in this case for glucose annunciations.